Source/UIKit additions/SVGKFastImageView.m

```diff
@@ -9,6 +9,8 @@
 @interface SVGKFastImageView ()
 @property(nonatomic,readwrite) NSTimeInterval timeIntervalForLastReRenderOfSVGFromMemory;
 @property (nonatomic, retain) NSDate* startRenderTime, * endRenderTime; /*< for debugging, lets you know how long it took to add/generate the CALayer (may have been cached! Only SVGKImage knows true times) */
+@property (nonatomic) BOOL didResigesterObservers, didResigesterInternalRedrawObservers;
```

Comment:
Please fix the typo: "didResigesterObservers" should be: "didRegesterObservers"

"didResigesterInternalRedrawObservers" should be "didRegesterInternalRedrawObservers"